### ISSUE TITLE
[A11Y] Fixed: Issues and Confusion with "You are here" screen reader heading for breadcrumbs.

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#5269]
## Issues and Confusion with "You are here" screen reader heading for breadcrumbs

**Description of Issue**: Breadcrumb navigation aria-labelledby name of "you are here" is confusing to screen reader users. They hear "You are here navigation" which sounds strange out of context.
**URL**:  https://pr3750-llnmetezlaix5oe10ddq5m3nya7zdz2b.tugboat.qa/exerci
**Testing Environment/Browser**: Windows Lenovo PC, Windows 10 Pro, Chrome Browser v. 94.0.4606.61, JAWS 2020.12.13
**[Severity](https://github.com/backdrop/backdrop-issues/issues/5244#issuecomment-931739260)**: P3
**Solution**: 

- in `core/includes/theme.inc`, line 2021
  1. Add an `aria-label="Website Orientation"` attribute to the `<nav>` element.
  1. Remove the `h2` containing `You are here`.
- in `core/modules/system/system.module`:, line 3274
  1. Add an `aria-current="page"` attribute to the `'<span>'` element for the current page.

A more intuitive aria-label might be: "Website Orientation." So a screen reader user would hear "Website orientation navigation"

For the breadcrumb that indicates the current page, the following ARIA attribute should be added: aria-current="page"

Code recommendation:
```html
<nav class="breadcrumb" aria-label="Website Orientation">
  <ol>
       <li><a href="/"> Home </a></li>
        ......
       <li><span aria-current="page"> Foo </span></li>
  </ol>
</nav>
```

## Documentation
* https://docs.backdropcms.org/api/backdrop/core%21includes%21theme.inc/function/theme_breadcrumb/1 
* https://docs.backdropcms.org/api/backdrop/core%21modules%21system%21system.module/function/system_breadcrumb_block/1

## Screenshot:
![image](https://user-images.githubusercontent.com/12848168/135536988-a2093741-44e1-4539-ba0b-c5ae5911aba7.png)

**Comment by akadey** [comment]
I would like to work on this issue!

**Comment by developer-diganta** [comment]
Hello! Is this issue still open? Then I would like to work on it.

**Comment by stpaultim** [comment]
@developer-diganta - Yes, this issue is still open. The previous comment was from a prior hack-a-thon and the work was not completed. Feel free to work on this issue and to let us know if you have questions. 

You can sometimes reach me in the OpenForce2022 discord channel as well.

**Comment by developer-diganta** [comment]
@stpaultim in the system.module, where shall I make the changes? I did not get which element shall I add the aria-current to...

**Comment by klonos** [comment]
@developer-diganta I've updated the issue summary:

> 1. Add an `aria-current="page"` attribute to the `'<span>'` element for the current page.

**Comment by stpaultim** [comment]
@jenlampton or @klonos - Are either of you able to look at this? I'm not sure how to test this or if I understand the original instructions clear enough to tell if this is correct?

**Comment by klonos** [comment]
Code-wise this looks good 👍🏼  ...thanks @developer-diganta 🙏🏼 

I have tested and confirmed that the markup is as expected when using `theme_breadcrumb()`, but there are other places where the "You are here" text is being added:
- `core/themes/seven/template.php`
- `core/themes/basis/template.php`

We should fix those too.

**Comment by developer-diganta** [comment]
@klonos shall I do those two then?

**Comment by klonos** [comment]
Yes please @developer-diganta

**Comment by developer-diganta** [comment]
@klonos Shall I remove that and add `aria-label="Website Orientation"`?

**Comment by klonos** [comment]
@developer-diganta yes, the change in both theme templates should be similar to the changes you have already made in the `core/includes/theme.inc` file.

**Comment by developer-diganta** [comment]
@klonos Changed it!

**Comment by stpaultim** [comment]
@developer-diganta - Excellent. Thanks for your help! Feel free to work on another issue while we wait for feedback from @klonos or @jenlampton.

**Comment by indigoxela** [comment]
Sorry, I know I'm a little late here. The PR looks good, it's a great improvement, but the aria-labels should be made translatable.

**Comment by stpaultim** [comment]
@developer-diganta - There is some feedback on your PR. Are you available and interested to make the change? We have a release happening in a few days.

**Comment by developer-diganta** [comment]
@stpaultim Sure, I'll get it done by today...

**Comment by stpaultim** [comment]
@Thanks for the updated PR @developer-diganta

**Comment by indigoxela** [comment]
@developer-diganta great work, one `t()` too much, though. See my comment on your PR for more hints.

**Comment by indigoxela** [comment]
@developer-diganta many thanks for your prompt code update! The functional tests passed, this looks good to go for me now. :+1:

**Comment by jenlampton** [comment]
I've tested the PR and can conform the markup is now as recommended: https://pr3978-upd9ifltw5oyc3ejx8f3au9ntuewfjja.tugboatqa.com/about

Code review looks straightforward, thank you @developer-diganta for your work on this!

**Comment by laryn** [comment]
Thanks for filing this A11y issue @jenlampton, and for the work and persistence of making the PR @developer-diganta! Thanks for guidance, testing, and reviews @stpaultim, @indigoxela and @klonos! I've merged https://github.com/backdrop/backdrop/pull/3978 into 1.x and 1.22.x.
